compiler/src/main/scala/org/scalajs/core/compiler/ScalaJSPlugin.scala

```diff
@@ -53,6 +53,13 @@ class ScalaJSPlugin(val global: Global) extends NscPlugin {
     var absSourceMap: Option[URI] = None
   }
 
+  /** Checks and registers module exports on the symbol.
+   *  This bridge allows other plugins (such as ScalaJSJUnitPlugin) to register
+   *  new modules for export between PrepJSInterop and GenJSCode phases.
```

Comment:
The phases are called `jsinterop` and `jscode`. You might want to drop a line that this basically just reapplies some transformations. The calling code still needs to add `@JSExport` annotations.